R/sidebar_layout.R

```diff
@@ -0,0 +1,208 @@
+# ------------------------------------------------------------------------------
+# Sidebar layout composed of main and sidebar panels
+# ------------------------------------------------------------------------------
+#'
```

Comment:
also here I think that one comment on the top of the file with context is good, but without `----...` and it should be disattached from the comment that is part of the docs below, otherwise roxygen treats it as a part of this func. description.